charts/argo-rollouts/Chart.yaml

```diff
@@ -20,3 +20,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: Update to app version 1.5.1
+    - kind: fixed
+      description: Fix use prometheus metrics service
```

Comment:
Thank you for your PR! 
Can you please remove old changelog and add new line in the end of file?